recipe/meta.yaml

```diff
@@ -1,23 +1,23 @@
-{% set version = "20131217" %}
+{% set version = "20180427" %}
 
 package:
   name: x264
-  version: 20131217
+  version: {{ version }}
 
 source:
   fn: x264-snapshot-{{ version }}-2245-stable.tar.bz2
   url: http://download.videolan.org/pub/videolan/x264/snapshots/x264-snapshot-{{ version }}-2245-stable.tar.bz2
-  md5: cfd27222c7aa9983259ddb10f7f570a6
+  md5: 89fb34fe9d0959894592e308cafff5ad
 
 build:
-  number: 3
+  number: 0
   skip: true         # [win]
 
 requirements:
   build:
     - toolchain
     - perl 5.20.3.1
-    - yasm
+    - nasm
```

Comment:
Note to others: Upstream switched from `yasm` to `nasm`. ( https://stackoverflow.com/a/44573541 )